tests/pages/bigtest/package-show.js

```diff
@@ -32,6 +93,26 @@ import {
     name: text('[data-test-eholdings-title-list-item-title-name]'),
     selectedLabel: text('[data-test-eholdings-title-list-item-title-selected]')
   });
+
+  hasCustomCoverage = isPresent('[data-test-eholdings-package-details-custom-coverage-display]')
+  customCoverage = text('[data-test-eholdings-package-details-custom-coverage-display]')
+  hasCustomCoverageAddButton = isPresent('[data-test-eholdings-package-details-custom-coverage-button] button')
+  clickCustomCoverageAddButton = clickable('[data-test-eholdings-package-details-custom-coverage-button] button')
+  clickCustomCoverageCancelButton = clickable('[data-test-eholdings-package-details-cancel-custom-coverage-button] button')
+  clickCustomCoverageEditButton = clickable('[data-test-eholdings-package-details-edit-custom-coverage-button] button')
+  clickCustomCoverageSaveButton = clickable('[data-test-eholdings-package-details-save-custom-coverage-button] button')
+  isCustomCoverageDisabled = property('disabled', '[data-test-eholdings-package-details-save-custom-coverage-button] button')
+  validationError = text('[data-test-eholdings-custom-coverage-date-range-begin] [class^="feedbackError"]')
+
+  dateFields = new PackageDatePicker()
```

Comment:
Should this be scoped to something? 

i.e. `new PackageDatePicker('[data-test-eholdings-package-details-custom-coverage]')`

Reply:
Yes it should. I also made it plural to make it clear it's for both date pickers. It could use another refactoring pass in the future to make a page object for a generic datepicker. That would DRY this up a lot.